### ISSUE TITLE
 Application problems

### ISSUE THREAD

**Issue by L-max-min** [michaelrzhang/lookahead#7]
Hello, why can't I apply your lookahead-TensorFlow to my network? My application is as follows, may I ask how to solve it? Besides, could you please tell me what ARGS is

from lookahead02 import Lookahead
optimizer =tf.keras.optimizers.Adam(learning_rate=1e-3)
model.compile(optimizer=Lookahead(optimizer, la_steps=5, la_alpha=0.8),loss='mse')

Model. fit(train_X, train_y, epochs=50, batch_size=72,verbose=2) the parameter var_list is missing，please，can you tell me what should var_list is and be set to
images.githubusercontent.com/95292409/163957772-f5c88a9f-5ec6-44fe-ad4d-3d0847ad1a60.png)

**Comment by zaccharieramzi** [comment]
You can try using the TensorFlow/Keras official implementation: https://www.tensorflow.org/addons/api_docs/python/tfa/optimizers/Lookahead

I haven't used it yet so not sure it will work, but it might just do the trick.

**Comment by michaelrzhang** [comment]
Yes, the version in Tensorflow should work!
